packages/node_modules/@ciscospark/react-component-utils/src/badge.js

```diff
@@ -0,0 +1,142 @@
+import * as NotificationsConstants from './constants';
+
+/**
+ * Get the badge state of a conversation
+ * @param  {object} space - a space
+ * @param  {boolean} [unread=false] - whether this conversation has unread activities
+ * @param  {boolean} [hasMention=false] - whether this conversation has unread mentions
+ * @param  {string} [globalNotificationState=NotificationsConstants.NOTIFICATIONS_ALL]
+ *          - the value of global notification setting
+ * @returns {string} a value that indicates what the badge of this conversation should be.
+ */
+export function getBadgeState(space, unread = false, hasMention = false,
```

Comment:
Let's convert these to an object parameter instead of multiple parameters. So `getBadgeState({space, unread = false` etc.